### PR TITLE
S1t6   create job card component

### PR DESCRIPTION
**Pull request by maxwell-01**

### CODE REVIEW

File: src/CompanyLogo/CompanyLogo.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const CompanyLogo = (props) => {
+
+    return (
+        <img className={'img-fluid'} src={props.logo} alt={'logo for ' + props.company} />
+    )
+
```
Comment (oritchie77) [inline src/CompanyLogo/CompanyLogo.js]: remove extra lines here please

File: src/JobTable/JobTable.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                    <th className="col-4" scope="col">Skills</th>
+                </tr>
+                </thead>
+                <tbody>
+                {(recentJobs != null) ? (
+                        recentJobs.map(job => (
+                            <JobCard job={job} key={job.id}/>
+                        ))
+                ) : (
+                    <td>Loading....</td>
```
Comment (oritchie77) [inline src/JobTable/JobTable.js:35]: is this to tell the user that there are now jobs? might be worth something like no job found matching criteria

File: src/Type/Type.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const Type = (props) => {
+
+    return (
+        <>
+            {(props.type != null) ? <span className={'badge rounded-pill bg-primary'}>{props.type}</span> : ''}
+        </>
+    )
+
```
Comment (oritchie77) [inline src/Type/Type.js]: remove extra lines

**Review by oritchie77 — CHANGES_REQUESTED** [review]
changes as stated below, but cracking work guys

**Comment by maxwell-01** [comment]
changes made in line with PR comments

**Review by oritchie77 — APPROVED** [review]

**Pull request merged**